agents/ibootbar/ibootbar.py

```diff
@@ -461,15 +350,17 @@ def reboot(self, session, params=None):
                    respective initial states. This tasks takes about 30 seconds.
         """
         with self.lock.acquire_timeout(3, job='reboot') as acquired:
-            if acquired:
-                # Send set command to set rebootSystem to True
-                reboot = [('IBOOTPDU-MIB', 'rebootSystem', 0)]
-                setcmd = yield self.ibootbar.snmp.set(reboot, self.ibootbar.version, 1)
-                print(setcmd)
-
-                return True, 'Rebooting system. This will take 30 seconds.'
-            else:
+            if not acquired:
                 return False, "Could not acquire lock"
+            # Send SNMP SET command to set rebootSystem to True
+            reboot = [('IBOOTPDU-MIB', 'rebootSystem', 0)]
+            setcmd = yield self.snmp.set(reboot, self.version, 1)
+            self.log.info('{}'.format(setcmd))
+
+        # Force SNMP GET status commands
+        self.lastGet = self.lastGet - 60
```

Comment:
Related to my question in the first review: https://github.com/simonsobs/socs/pull/306#discussion_r905217711

Does this result in all outlets turning "off" in the acq process during the reboot?

Reply:
Renaming the task sounds good. Users might interpret "reboot" as "cycle all outlets". I'm not really sure we need a "cycle all outlets" task at the moment, so let's just stick to renaming this. If we find we need that we can add it later.